### ISSUE TITLE
Custom Control Panel Issues

### ISSUE THREAD

**Issue by HoraceAndTheSpider** [midwan/amiberry#822]
**Scenario**
- Amiberry v4.1.1
- Tested on self-compiled copy for Raspberry Pi 3
- Loading games via WHDBooter (through RetroPie)

**Issue 1**
UAE Config has custom control set in it, but are not loaded on booting. Other options from the Config seem to be applied OK.

**Issue 2**
"Menu Button" is shown as an option, but my controller neither has one, nor is it mapped from the Retroarch controller config. Retroarch will use a combination (Hotkey+Something) for its actions, which are still shown OK.

This button / option should be 'greyed out' when not applicable.

**Issue 3**
Layout of the custom control panel is not longer aligned to the physical layout of a pad; this is now less intuitive for users.

Previously this was arranged with the D-Pad on the left of the panel, plus left analogue button & shoulder button, and on the right side of the panel, the Fire Buttons plus right analogue button * shoulder button

**Comment by midwan** [comment]
The custom controls not being loaded was fixed with fd462b21856e256550a0b9f2616543e9ed8bb44f
Please test with the latest version (currently v4.1.4) to verify this.

**Comment by HoraceAndTheSpider** [comment]
As per Discord chat - confirmed on v4.1.4

**Comment by midwan** [comment]
1) Can't recreate that problem here, at least after the latest updates committed today. I can save custom control settings on a controller button to a config, and it loads (and shows) correctly. Pressing that button performs the assigned function (I tested Quit Emulator). Perhaps it was fixed as part of the other improvements?
2) This is coming from SDL2 by iterating all controller buttons, it's the "GUIDE" button that some controllers have (e.g. PS4 ones have one with the PS symbol, XBox ones have their own, etc.). Not sure if I can disable it if not available, we'll see.
3) This was done on purpose to clean up and simplify the handling of all the possible buttons there, which can be dynamically increased or reduced depending on the version of SDL2. Newer versions support more buttons that can be mapped from controllers. I'm thinking of an overhaul in this panel in general, with a more graphical layout (perhaps an image of a controller) and with a full remap function available - but this will come later, as it requires some work to do.
